app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance" | int) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: "Distance" | int) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, other: "Distance" | int) -> "Distance":
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: "Distance" | int | float) -> "Distance":
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: int) -> bool:
```

Comment:
handle cases when other is an instance of Distance class. in the following methods as well